onyo/lib/commands.py

```diff
@@ -70,31 +74,29 @@ def wrapper(*args: P.args, **kwargs: P.kwargs) -> T:
 
 def fsck(repo: OnyoRepo,
          tests: list[str] | None = None) -> None:
-    r"""Run a suite of checks to verify the integrity and validity of an Onyo
-    repository and its contents.
+    r"""Run a suite of integrity checks on an Onyo repository and its contents.
 
-    By default, the following tests will be performed:
+    By default, the following tests are performed:
 
-    * ``clean-tree``: verify that git has no changed (staged or unstaged) or
-      untracked files
     * ``anchors``: verify that all directories (outside of .onyo) have an
```

Comment:
`.onyo/` and all its subdirectories have anchor files too. No idea if they are checked for, this might be a bug, but I find this sentence to not reflect the reality of my onyo repository :)

Also, the path `.onyo` should probably be in backticks or double backticks even.